core-web/libs/contenttype-fields/tsconfig.json

```diff
@@ -1,14 +1,5 @@
 {
-    "compilerOptions": {
```

Comment:
why delete these, is part of the `tsconfig.base.json` now ?

Reply:
Yes! Now it's part of the `tsconfig.base.json`.